draft-ietf-ppm-dap.md

```diff
@@ -758,7 +748,7 @@ of the Aggregators is configured with following parameters:
   requirements are described in {{verification-key}}.
 
 Finally, the Collector is configured with the HPKE secret key corresponding to
-`collector_hpke_config`.
+`collector_config`.
```

Comment:
FWIW I wish this were called "collector_hkpe_config" ("collector_config" is a bit vague)

Reply:
I don't remember why I made this change, and no disagreement here that `collector_hpke_config` is easier to understand. I switched back.